src/helpers.ts

```diff
@@ -65,3 +65,7 @@ export const formatMessage = (guild: Discord.Guild) => (
 
 	return output.trim();
 };
+
+export const mapToArray = <T>(map: Collection<string, T>): Array<T> => {
```

Comment:
1. One of the limitations of Redux is that it doesn't play well with non-serializable data. Although it can be coerced into working with maps by providing a custom serialization function, it doesn't seem possible to use with toolkit (which injects immer so you can just mutate state in reducers)
2. I removed it as there were better methods in the discordjs collection